decidim-department_admin.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,db,config,lib}/**/*", "LICENSE-AGPLv3.txt", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
+  s.add_dependency "decidim-conferences", DECIDIM_VER
```

Comment:
This dependency is optional, it probably should be an `add_development_dependency` instead. We should not obligate all users of `department_admin` to install `decidim-conferences`.